usaspending_api/etl/management/commands/es_rapidloader.py

```diff
@@ -161,6 +178,11 @@ def controller(self):
         download_proccess.join()
         es_index_process.join()
 
+        if self.config['recreate']:
```

Comment:
if self.config['recreate'] is `true`, won't it keep deleting the single destination index?

Reply:
recreate is default set to false if flag isn't there. no, there are no deletes, it would close index patterns of previous_index. no deletes until we have a snapshot. someone would need to run this explicitly with --r or -recreate for indices to be closed

Reply:
I verified locally using a single FY of data. If the `-r` flag is provide the script will continually delete the only index and recreate it for each file download aka "job". This will ultimately lead to the index with only data from the final job.